### PR TITLE
[P4-1185] Updates default fallback question for building personal care need

### PR DESCRIPTION
**Pull request by willfish**
### Jira link

P4-1185

### What?

The initial implementation of the AssessmentAnswer documents for personal care needs was only for female prisons where the person might be pregnant.

This updates the default question to be more appropriate for a wider gamut of people we might move.

### Why?

This is principally a UX change that will make using our service make more sense.

### CODE REVIEW

File: spec/services/moves/import_people_spec.rb [hunk 1]
```diff
@@ -27,16 +27,16 @@
      { prison_number: prisoner_two.nomis_prison_number, first_name: 'Bob' }]
   end
   let(:personal_care_needs_response) do
-    [{ offender_no: prisoner_one.nomis_prison_number, problem_type: 'MATSTAT', problem_code: 'ACCU9' },
-     { offender_no: prisoner_two.nomis_prison_number, problem_type: 'MATSTAT', problem_code: 'ACCU9' },
-     { offender_no: prisoner_two.nomis_prison_number, problem_type: 'MATSTAT', problem_code: 'ACCU4' }]
+    [{ offender_no: prisoner_one.nomis_prison_number, problem_type: 'FOO', problem_code: 'AA' },
```
Comment (willfish) [inline spec/services/moves/import_people_spec.rb:30]: These keys plumb together the correct fallback. Previously the key returned for the problem code happened to be pregnancy by accident.

Comment (alexdesi) [reply spec/services/moves/import_people_spec.rb:30]: it's not clear from the test why this is returning `FOO` as problem_type,
does it means that it is a problem type we are not handling?

Comment (alexdesi) [reply spec/services/moves/import_people_spec.rb:30]: maybe you want just to add a context to make it clear:

context "when the response has problem_type which ...." do

end
or anything that can clarify that `problem_type: 'FOO'`

Comment (willfish) [reply spec/services/moves/import_people_spec.rb:30]: Yeah, I use foo because internally this value isn't used to control the behaviour of anything

Comment (willfish) [reply spec/services/moves/import_people_spec.rb:30]: I can probably just remove the problem type attribute

**Pull request merged**